src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -195,15 +231,25 @@ class Results extends React.Component<Props, State> {
                         query={eventView.getEventsAPIPayload(location).query}
                         organization={organization}
                         showLegend
-                        yAxisOptions={yAxisOptions}
-                        yAxisValue={eventView.yAxis}
-                        onYAxisChange={this.handleYAxisChange}
+                        yAxis={yAxisValue}
                         project={eventView.project as number[]}
                         environment={eventView.environment as string[]}
                       />
                     ),
                     fixed: 'events chart',
                   })}
+                  <ChartControls>
+                    <InlineContainer>
+                      <SectionHeading>{t('Count')}</SectionHeading>
+                      {totalValues === null ? '-' : <Count value={Number(totalValues)} />}
```

Comment:
Typescript complained until this `Number()` cast was added. I would have thought the `=== null` would let it narrow the value, but that's not how it worked out.